lib/atm.rb

```diff
@@ -7,14 +8,19 @@ class Atm
 
    def initialize
     @funds = 1000
+
+    #@balance = 1500
    end
 
+#ACCOUNT = Account.new
+
    def withdraw(amount, pin, expiry, account)
       case
       when negative_value(amount) then return_error_message(:negative_amount)
       when !check_pin(pin, account.pin) then return_error_message(:wrong_pin)
       when check_expiry(expiry, account.exp_date) then return_error_message(:card_expired)
       when !sufficient_fund_in_atm(amount) then return_error_message (:no_sufficient_fund)
+      when !balance_in_account(amount, account.balance) then return_error_message (:no_sufficient_balance)
```

Comment:
Lint/ParenthesesAsGroupedExpression: `(...)` interpreted as grouped expression. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)<br>Metrics/LineLength: Line is too long. [106/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/RedundantParentheses: Don't use parentheses around a literal.